src/test/java/mate/academy/hibernate/relations/dao/AbstractTest.java

```diff
@@ -4,6 +4,7 @@
 import java.util.List;
 import java.util.Properties;
 import javax.sql.DataSource;
+
```

Comment:
Is it necessary to make changes in this file?

Reply:
Still exists.